src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs

```diff
@@ -40,6 +41,64 @@ public async Task InsertMutation(string dbQuery)
             SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
         }
 
+        /// <summary>
+        /// <code>Do: </code> Inserts new row and return all its columns
+        /// <code>Check: </code> A row is inserted in the table that has rows with default values as built_in methods.
+        /// it should insert it correctly with default values correctly handled by database.
+        /// </summary>
+        public async Task InsertMutationWithDefaultBuiltInFunctions(string dbQuery)
```

Comment:
It's not clear which columns have default values and whether those values are `('string')``, `((1))` (int), or `((myFunc()))` which are functions

Reply:
how was this resolved?

Reply:
I have added in the test summary highlighting the default columns value